proto/pdpb.proto

```diff
@@ -433,6 +433,10 @@ message SplitRegion {
     repeated bytes keys = 2;
 }
 
+message SwitchWitness {
+    metapb.Peer peer = 1;
```

Comment:
Seems overwhelming to use `Peer`. How about using ID and flag?

Reply:
Sure, using ID and flag instead, thanks!